### ISSUE TITLE
Improving other noisy data instead of Voice bank

### ISSUE THREAD

**Issue by asadullah797** [deNsuh/segan-pytorch#28]
Hi,
I have used GAN for speech enhancement using AMI speech dataset but it does not improve the noisy far-field speech data samples. Although it works well on Voice bank dataset as mentioned in the paper.
Can you please give any comments.
Thanks

**Comment by dansuh17** [comment]
Sorry for the late reply. Haven't experimented with AMI dataset. Perhaps the far-field speech data samples have low SNRs than VCTK's?

**Comment by asadullah797** [comment]
Thank you for your reply.
If I mix noises to the librispeech and consider librispeech as clean data and noisy librispeech as noisy data. Will this GAN work in that case?

**Comment by dansuh17** [comment]
Yes. There is no specific reason for this model to not work for other datasets than VCTK, since it includes no VCTK-specific treatments to work.
